werk.txt

```diff
@@ -1,2 +1,5 @@
 WERK
-MORE WORK
\ No newline at end of file
+MORE WORK
+
+
+My work, That i did
```

Comment:
```suggestion
MORE WORK
My work, That i did
```
You said "added a line" that is singular. So this would be more like the PR title.